### PR TITLE
test: add unit tests to textinput component

### PR DESCRIPTION
**Pull request by jenifersantoszup**
### Related Issues
#1202

### Description and Example

Unit tests were created for the textinput component.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/TextInputTest.kt [hunk 1]
```diff
@@ -49,8 +50,8 @@ const val HIDDEN = true
 const val STYLE_ID = "Style"
 val TYPE = TextInputType.NUMBER
 
-@DisplayName("Given Text Input")
-class TextInputTest : BaseComponentTest() {
+@DisplayName("Given a TextInput")
```
Comment (uziasferreirazup) [inline android/beagle/src/test/java/br/com/zup/beagle/android/components/TextInputTest.kt:53]: Please adjust to the correct struct has in the style guide: https://github.com/ZupIT/beagle/wiki/Android-Tests-Style-Guide

**Review by uziasferreirazup — DISMISSED** [review]

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/TextInputTest.kt (hunk 1 above)
Comment (uziasferreirazup) [reply android/beagle/src/test/java/br/com/zup/beagle/android/components/TextInputTest.kt:53]: in this case you have Given and then but missing the `When`

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
Then the component should be render correctly | br.com.zup.beagle.android.engine.renderer.ViewRendererFactoryTest | 0.04|


<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by uziasferreirazup** [comment]
please, run the test with coverage to check what needs to add a unit test to missing things

**Comment by uziasferreirazup** [comment]
Some test you can add test to listeners setUpOnFocusChange, setUpOnTextChange

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
Then the component should be render correctly | br.com.zup.beagle.android.engine.renderer.ViewRendererFactoryTest | 0.083|


<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by gabrielleonardozup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Review by carlossteinzup — APPROVED** [review]

**Review by paulomeurerzup — APPROVED** [review]

**Review by hthemir — APPROVED** [review]

**Review by gabrielleonardozup — APPROVED** [review]

**Pull request merged**